modules/drivers/lidar_surestar/BUILD

```diff
@@ -0,0 +1,53 @@
+load("//tools:cpplint.bzl", "cpplint")
+
+package(default_visibility = ["//visibility:public"])
+
+cc_binary(
+    name = "libsurestar_driver_component.so",
+    linkopts = ["-shared"],
+    linkstatic = False,
+    deps = [":driver"],
+)
+
+cc_library(
+    name = "driver",
+    srcs =  [
+        "src/component/component.cpp",
+        "src/driver/driver.cpp",
+        "src/driver/driver16.cpp",
```

Comment:
Split the big target into smaller ones.

Reply:
I use hesai driver as a reference which applies its driver into one target. Do you want me to split the target into what scale? Is there any rules I can follow?

Reply:
See the link below on Best Practice for BUILD file.

https://docs.bazel.build/versions/master/bazel-and-cpp.html#build-files

The `hesai` driver you mentioned should also be formatted by our team in the near future. And relative docs on contributing should be updated. Sorry for the inconvenience. And hope we can work together to make Apollo easier to read/use/maintain. Thank you!

Reply:
I have just updated the hesai driver to conform to Bazel Best Practice. Refer to #12349 for more.

Reply:
changed